src/driver.rs

```diff
@@ -256,11 +256,14 @@ pub fn main() {
     LazyLock::force(&ICE_HOOK);
     exit(rustc_driver::catch_with_exit_code(move || {
         let mut orig_args: Vec<String> = env::args().collect();
+        let has_sysroot_arg = arg_value(&orig_args, "--sysroot", |_| true).is_some();
```

Comment:
Note that if we *do* want to support passing --sysroot to clippy-driver (with `--` to cargo-clippy, not rustflags), this will need to be changed to look at `clippy_args` as well, not just `orig_args`.

Reply:
Could I send a PR to support passing `--sysroot` to `clippy-driver`?